pyproject.toml

```diff
@@ -36,7 +36,6 @@ dependencies = [
   "jinja2",
   "jsonschema >=4.19",
   "libarchive-c",
-  "lief",
```

Comment:
```
# "lief",  # see https://github.com/conda/conda-build/issues/5101
```

Reply:
I expect the canary section of this CI build to run the pip check test.

Reply:
canaries are only built for PRs if the https://github.com/conda/conda-build/labels/build%3A%3Areview label is applied